Example/SnapshotTests/BPKCalendarSnapshotTest.m

```diff
@@ -60,11 +61,11 @@ - (void)configureParentView:(UIView *)parentView forCalendar:(BPKCalendar *)cale
                                                                        options:0
                                                                        metrics:nil
                                                                          views:@{@"calendar": calendar}]];
-    [parentView addConstraints:[NSLayoutConstraint constraintsWithVisualFormat:@"[parentView(320)]"
```

Comment:
Does this mean the current calendar design will not work on the previous generation of iPhone SE?

Reply:
You mean for price labels?
I guess if the iPhone SE has a screen width of 320 then no 🤷

Reply:
Okay that's probably worth highlighting too

Reply:
Do we still support iOS 12?

Reply:
iOS 13 supports the SE 😮

Reply:
Some iPod is also 320pt wide I believe